docs/apis-tools/zeebe-api/gateway-service.md

```diff
@@ -10,6 +10,31 @@ The Zeebe client gRPC API is exposed through a single gateway service. The curre
 can be found in
 the [Zeebe repository](https://github.com/camunda/camunda/blob/main/zeebe/gateway-protocol/src/main/proto/gateway.proto).
 
+## Default Service Config
+
+Along with the gateway protocol definition, the gateway service also bundles a [default service configuration file](https://github.com/camunda/camunda/blob/main/zeebe/gateway-protocol-impl/src/main/resources/gateway-service-config.json).
+This file can be used as is, or as template to create your own, and defines default retry strategies on a per RPC basis: when to retry (based on error code), how often, how soon, etc.
+This file is also loaded by the Camunda Java SDK if `useDefaultRetryPolicy` is set to true.
```

Comment:
Perhaps link to the docs on the SDK here?

Reply:
@npepinpe, Christina makes a good point here, but do you mean the Spring Zeebe SDK or the Java Client? Or something else?

Reply:
Well, both will do this, I guess... :thinking:

Reply:
We can always note that both offer this functionality, unless we want to direct the user toward a particular experience 🤷‍♀️

Reply:
Alright, I'll stick with Java client, because I don't think the setting is explicitly exposed in the Spring SDK (though you can still use it, it's just a bit more hidden).